### PR TITLE
Authorization implementation

### PR DESCRIPTION
**Pull request by callkalpa**
Samlpe JWT generation,

```
from jose import jwt

key = "jwt_secret"
payload = {
    'areaAssignment/dataEditor': [
        {
            "areaName": "1",
            "areaId": 7
        }
    ],
    'areaAssignment/ECLeadership': [
    ]
}
encoded = jwt.encode(payload, key)
print(encoded)
```

### CODE REVIEW

File: auth/__init__.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+from orm.entities.Submission import TallySheet
+from orm.entities.Submission.TallySheet import TallySheetModel
+
+JWT_SECRET = "jwt_secret"
+CLAIM_PREFIX = "areaAssignment/"
+AREA_ID = "areaId"
+
+
+def get_unauthorized_msg(detail):
+    return connexion.problem(401, "Unauthorized", detail)
```
Comment (dinukadesilva) [inline auth/__init__.py]: There's an exception class I have defined. May be we could use that

**Review by dinukadesilva — CHANGES_REQUESTED** [review]
Shall we add this code may be to readme. Or add this as a script file to the repo. So, that it's documented. 

I would +1  for adding to readme

File: auth/__init__.py (hunk 1 above)
Comment (callkalpa) [reply auth/__init__.py]: Fix with 066207d

**Comment by callkalpa** [comment]
@dinukadesilva, sample JWT generation code snippet is moved to README with 7a535e5.

**Review by dinukadesilva — APPROVED** [review]

**Comment by dinukadesilva** [comment]
@callkalpa 

Im fixing the merge conflicts.

And I'm adding the token generation code to index.py. SO, it comes straight way when the app is started.

**Pull request merged**